docs/features/ethernet-lan.md

```diff
@@ -39,5 +40,22 @@ ESP32Deux |  | 17, 18, 23
 [WESP32](https://wesp32.com/) |  | 0, 16, 17
 [WT32-ETH01](https://www.seeedstudio.com/Ethernet-module-based-on-ESP32-series-WT32-ETH01-p-4736.html) |  | 0, 16, 18, 23
 
+## Defining GPIO for a custom PCBA
 
+The following PHY are supported by WLED according to the header ETH.h:
```

Comment:
ETH.h is not part of WLED and WLED does not support anything but LAN8720A and IP101.